src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -1237,16 +1247,81 @@ impl A320HydraulicBrakingLogic {
 
 impl SimulationElement for A320HydraulicBrakingLogic {
     fn read(&mut self, state: &mut SimulatorReader) {
-        self.parking_brake_demand = state.read_bool("BRAKE PARKING INDICATOR");
+        self.parking_brake_demand = state.read_bool("BRAKE PARKING POSITION");
         self.weight_on_wheels = state.read_bool("SIM ON GROUND");
         self.is_gear_lever_down = state.read_bool("GEAR HANDLE POSITION");
         self.anti_skid_activated = state.read_bool("ANTISKID BRAKES ACTIVE");
-        self.left_brake_pilot_input = state.read_f64("BRAKE LEFT POSITION") / 100.0;
-        self.right_brake_pilot_input = state.read_f64("BRAKE RIGHT POSITION") / 100.0;
+        self.left_brake_pilot_input = state.read_f64("BRAKE LEFT POSITION");
+        self.right_brake_pilot_input = state.read_f64("BRAKE RIGHT POSITION");
         self.autobrakes_setting = state.read_f64("AUTOBRAKES SETTING").floor() as u8;
     }
 }
 
+struct A320BrakingForceSimulationOutput {
+    left_brake_pedal_position: f64,
+    right_brake_pedal_position: f64,
+
+    left_brake_force: f64,
+    right_brake_force: f64,
+
+    park_brake_lever_is_set: bool,
+}
+impl A320BrakingForceSimulationOutput {
+    pub fn new() -> Self {
+        A320BrakingForceSimulationOutput {
+            left_brake_pedal_position: 0.,
+            right_brake_pedal_position: 0.,
+
+            left_brake_force: 0.,
+            right_brake_force: 0.,
+
+            park_brake_lever_is_set: true,
+        }
+    }
+
+    pub fn update_forces(
+        &mut self,
+        norm_brakes: &BrakeCircuit,
+        altn_brakes: &BrakeCircuit,
+        brake_logic: &A320HydraulicBrakingLogic,
+    ) {
+        // Updating raw pilots inputs just to update pedals animation in the sim
+        self.left_brake_pedal_position = brake_logic.left_brake_pilot_input;
+        self.right_brake_pedal_position = brake_logic.right_brake_pilot_input;
+
+        let left_force_norm = norm_brakes.left_brake_pressure().get::<psi>() / 3000.;
+        let left_force_altn = altn_brakes.left_brake_pressure().get::<psi>() / 3000.;
+        self.left_brake_force = left_force_norm + left_force_altn;
+        self.left_brake_force = self.left_brake_force.max(0.).min(1.);
+
+        let right_force_norm = norm_brakes.right_brake_pressure().get::<psi>() / 3000.;
+        let right_force_altn = altn_brakes.right_brake_pressure().get::<psi>() / 3000.;
```

Comment:
**suggestion**: consider adding a constant indicating what `3000` represents. As this value might be repeated in other places you could consider putting the constant outside of this function.